### DISCUSSION TITLE
can't use windhawk taskbar size mod: spawn g++.exe ENOENT

### DISCUSSION THREAD

**Discussion by rbutera** [ramensoftware/windhawk#161] [Q&A]
![image](https://github.com/ramensoftware/windhawk/assets/6047293/4fb52cd3-e284-455a-9275-d82d4bef0bb6)

fresh install of windhawk (straight from the website). latest windows 11. 

Whenever I try to install and run the taskbar size tweak I get an error:

```
spawn c:\Program Files\Windhawk\Compiler\bin\g++.exe ENOENT
```

any ideas? I have no idea where to download the g++ compiler to put in the bin folder. Is this in the documentation?

**Comment by m417z** [comment]
The `c:\Program Files\Windhawk\Compiler\bin\g++.exe` file is part of Windhawk, and it seems to be missing on your computer. Perhaps it was removed by mistake, or removed by an antivirus or a similar program. Try uninstalling and re-installing Windhawk. Settings will be kept (unless you choose to remove all data when uninstalling).

**Reply by rbutera** [reply]
I tried uninstalling windhawk and reinstalling and I'm having the same issue.

**Reply by m417z** [reply]
And the `c:\Program Files\Windhawk\Compiler\bin\g++.exe` file is missing, right? You need to find out why it's being deleted. I'd start by checking the antivirus logs.

**Comment by rbutera** [comment]
I've looked through antivirus logs and there's nothing. 

I built a new windows pc today and installed windhawk, and everything works fine on that computer.

On my old pc, nothing I try can make that g++ file appear. I guess I could copy paste it from machine to machine

**Reply by m417z** [reply]
You can try and see if it works. Also worth checking if other files are missing.

**Comment by Bigbobbie27** [comment]
Hi, mate.

Now, i still faced this issue. Could you please share how to fix it. The G++ still not found and I can't install any mod.

![image](https://github.com/user-attachments/assets/0fb7c4aa-ac0a-4fda-b3de-9f7751bb7eff)

**Reply by m417z** [reply]
Did you read my other comment here? Have you tried reinstalling Windhawk? Are you using an antivirus?
